src/main/java/reposense/util/TimeUtil.java

```diff
@@ -66,118 +63,61 @@ public static String getElapsedTimeMessage() {
     }
 
     /**
-     * Get the raw offset in milliseconds for the {@code zoneId} timezone compared to UTC.
+     * Returns a {@code LocalDateTime} that is set to midnight.
      */
-    public static int getZoneRawOffset(ZoneId zoneId) {
-        Instant now = Instant.now();
-        ZoneOffset zoneOffset = zoneId.getRules().getOffset(now);
-        return zoneOffset.getTotalSeconds() * 1000;
-    }
-
-    /**
-     * Get the {@code current} date that is in the system's timezone into the {@code zoneId} timezone.
-     */
-    public static Date getZonedDateFromSystemDate(Date current, ZoneId zoneId) {
-        int zoneRawOffset = getZoneRawOffset(zoneId);
-        int systemRawOffset = getZoneRawOffset(ZoneId.systemDefault());
-        return new Date(current.getTime() + zoneRawOffset - systemRawOffset);
-    }
-
-    /**
-     * Returns a {@code Date} that is set to midnight for the given {@code zoneId}.
-     */
-    public static Date getZonedSinceDate(Date sinceDate, ZoneId zoneId) {
+    public static LocalDateTime getZonedSinceDate(LocalDateTime sinceDate) {
```

Comment:
It is no longer timezone-aware, so the method name should be changed to avoid confusion.